src/libs/ajax.js

```diff
@@ -56,6 +57,7 @@ const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
   const somePointInTheFuture = Date.now() + timeout
   const maxDelayIncrement = 1500
   const minDelay = 500
+  //const captureFailure = _.once(() => captureRetryFailure(...args))
```

Comment:
Do we still need the commented out method (and call further down?)

Reply:
It was just there for personal convenience if I needed to reinstate it back into withRetryOnError(), took it out now.